controls/acl_exists.rb

```diff
@@ -0,0 +1,17 @@
+title 'Tests to confirm acl exists'
+
+plan_name = input('plan_name', value: 'acl')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+acl_relative_path = input('command_path', value: '/bin/getfacl')
+acl_installation_directory = command("hab pkg path #{plan_ident}")
```

Comment:
If this command fails it would be tough to understand what happened, similar to below you could check on this e.g.
```
  describe acl_installation_directory do
    its('exit_status') { should eq 0 }
    its('stdout') { should_not be_empty }
  end
```

Reply:
adding same installation directory check to both the test controls (*_exists and *_works)